problems/1.intro.js

```diff
@@ -14,33 +15,40 @@ function helloWorld() {
  * @param {string} name - the name to be added to the console log 
  * @returns undefined
  */
-function helloName() {
 
+function helloName(name) {
+  return "hello" + name;
 }
+console.log(helloName());
```

Comment:
Don't forget, if you're testing your function, you'll need to pass in the argument.